src/components/SelectDates/utils.js

```diff
@@ -0,0 +1,16 @@
+const rangedSome = (arr, predicate, start, end) => {
+  start = Math.max(start, 0)
+  end = Math.min(end, arr.length)
+  const step = start > end ? -1 : 1
+  if (!arr.length > 0) {
```

Comment:
This condition is weird. Why not `arr.length === 0` ?

Reply:
True